docs/real-device-deviceLogs.md

```diff
@@ -0,0 +1,117 @@
+---
+id: real-device-device-logs
+title: Device Logs
+hide_title: true
+sidebar_label: Device Logs
+description: With LambdaTest's Device Logs Feature, you can view the logs for the running app and the device during session. You can filter and download the logs for further analysis.
+
+keywords:
+- real device app testing 
+- lambdatest real device app testing
+- device logs
+
+url: https://www.lambdatest.com/support/docs/real-device-device-logs/
+site_name: LambdaTest
+slug: real-device-device-logs/
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import {YOUR_LAMBDATEST_USERNAME, YOUR_LAMBDATEST_ACCESS_KEY} from "@site/src/component/keys";
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "LambdaTest",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "Native Mobile App Testing",
+          "item": "https://www.lambdatest.com/support/docs/real-device-device-logs/"
+        }]
+      })
+    }}
+></script>
+
+# Device Logs feature on Real Devices
+
+The Device Logs Feature on Lambdatest Real Devices Platform empowers users to monitor both application and device logs conveniently. Through the Device Logs , you have the ability to sift through and download logs, facilitating thorough analysis of your application's behavior. These logs capture a detailed record of every event occurring during the App Live session, ranging from device selection and app installation to the activation of various device features.
+
+
+***
+
+## Accessing Device Logs
+
+**Step 1:** Login to your LambdaTest account. Go to **App Testing** under **Real Device** from the left sidebar.
+
+**Step 2**: Choose the OS version of your choice. ( Android or iOS )
+
+**Step 3**: Select Your Application and Device: Choose the desired application and device from the device listing to start your test session.
+
+**Step 4**: Manage **Device Logs**
+- Device logs are automatically captured from the start of the session.
+- You can open or close the Device Logs tab as per your requirements to view or hide the logs.
+
+<img loading="lazy" src={require('../assets/images/mobile-app-testing/devicelogs.webp').default} alt="Native Mobile App Testing"  className="doc_img" width="1366" height="629"/>
+
+
+## Features 
+
+1. **Filter Logs:**
+   - Utilize the filter bar to seamlessly filter logs based on entered text. Once text is entered, logs matching the filter criteria will be displayed, enabling focused analysis.
```

Comment:
This is not clear enough.

Once text is entered, only the logs matching the search text will be generated while keeping the existing logs as it is. This allows users to filter for specific logs for newly generated device logs.